tests/cli/plugin/plugin_tests.py

```diff
@@ -40,30 +40,59 @@ def tearDownClass(cls):
         TnsTest.tearDownClass()
         Folder.clean(cls.app_temp_path)
 
-    def test_100_plugin_add_after_platform_add_android(self):
-        result = Tns.plugin_add(plugin_name='tns-plugin', path=self.app_name)
-        assert 'Successfully installed plugin tns-plugin' in result.output
+    def test_100_plugin_add(self):
+        Tns.plugin_add(plugin_name='tns-plugin', path=self.app_name)
         assert File.exists(os.path.join(TnsPaths.get_app_node_modules_path(self.app_name), 'tns-plugin', 'index.js'))
```

Comment:
I think these two lines (45, 46) are  the same as line 51 and 52

Reply:
True, fixing it.
Just to note that the problem was existing before this PR.